front/app/translations/de-DE.json

```diff
@@ -1358,6 +1358,7 @@
   "app.errors.unparseable_excel": "Die ausgewählte Datei konnte nicht verarbeitet werden.",
   "app.errors.view_name_taken": "Eine Ansicht mit diesem Namen existiert bereits",
   "app.modules.commercial.flag_inappropriate_content.citizen.components.inappropriateContentAutoDetected": "Unangemessener Inhalt wurde automatisch in einem Beitrag oder Kommentar erkannt",
+  "app.modules.commercial.id_vienna_saml.components.continueWithStandardPortal": "Continue with StandardPortal",
```

Comment:
@nTraum doesn't this need to be translated?

Reply:
Yep, doing it right now.

Reply:
In [](https://github.com/CitizenLabDotCo/citizenlab/pull/1550/commits/22745e624612b71c07d244454c0f33a5854ba5e3)[22745e6](https://github.com/CitizenLabDotCo/citizenlab/pull/1550/commits/22745e624612b71c07d244454c0f33a5854ba5e3)

Reply:
Cool, I think you can hide the other languages so it doesn't get translated, but you would have to ask Karel about specifics.